lib/stack_master/commands/helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/concern'
+
+module StackMaster
+  module Commands
+    module Helper
```

Comment:
`Helper` is a pretty generic name. Perhaps rename to `TerminalHelper` or something more descriptive?